tests/translator/output/aws-cn/explicit_http_api.json

```diff
@@ -123,7 +123,7 @@
           },
           "openapi": "3.0.1",
           "paths": {
-            "$default": {
+            "/$default": {
```

Comment:
This shouldn't change. The template deploys just fine no?

Reply:
Ignore, just saw the property below.

Reply:
This test has FailedOnWarnings on.